app/src/main/java/com/framgia/forder/data/model/OrderHistory.java

```diff
@@ -0,0 +1,103 @@
+package com.framgia.forder.data.model;
+
+import com.framgia.forder.utils.Utils;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import static com.framgia.forder.utils.Constant.FORMAT_PRICE;
+import static com.framgia.forder.utils.Constant.UNIT_MONEY;
+import static com.framgia.forder.utils.Constant.VERTICAL_COLUMN;
+
+/**
+ * Created by levutantuan on 6/12/17.
+ */
+
+public class OrderHistory {
+    @Expose
+    @SerializedName("id")
+    private int mIdOrder;
+    @Expose
+    @SerializedName("quantity")
+    private int mQuantity;
+    @Expose
+    @SerializedName("price")
+    private double mPrice;
+    @Expose
+    @SerializedName("product_id")
+    private int mProductId;
+    @Expose
+    @SerializedName("created_at")
+    private String mTimeCreateOrder;
+    @Expose
+    @SerializedName("status")
+    private String mStatus;
+    @Expose
+    @SerializedName("name")
+    private String mNameProduct;
+
+    public int getIdOrder() {
+        return mIdOrder;
+    }
+
+    public void setIdOrder(int idOrder) {
+        mIdOrder = idOrder;
+    }
+
+    public int getQuantity() {
+        return mQuantity;
+    }
+
+    public void setQuantity(int quantity) {
+        mQuantity = quantity;
+    }
+
+    public double getPrice() {
+        return mPrice;
+    }
+
+    public void setPrice(double price) {
+        mPrice = price;
+    }
+
+    public int getProductId() {
+        return mProductId;
+    }
+
+    public void setProductId(int productId) {
+        mProductId = productId;
+    }
+
+    public String getTimeCreateOrder() {
+        return mTimeCreateOrder;
+    }
+
+    public void setTimeCreateOrder(String timeCreateOrder) {
+        mTimeCreateOrder = timeCreateOrder;
+    }
+
+    public String getStatus() {
+        return mStatus;
+    }
+
+    public void setStatus(String status) {
+        mStatus = status;
+    }
+
+    public String getNameProduct() {
+        return mNameProduct;
+    }
+
+    public void setNameProduct(String nameProduct) {
+        mNameProduct = nameProduct;
+    }
+
+    public String getPriceFormat() {
+        return String.format(FORMAT_PRICE, mPrice) + UNIT_MONEY;
```

Comment:
WARNING: [DefaultLocale] Implicitly using the default locale is a common source of bugs: Use `String.format(Locale, ...)` instead
 Calling `String#toLowerCase()` or `#toUpperCase()` *without specifying an explicit locale* is a common source of bugs. The reason for that is that those methods will use the current locale on the user's device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is *not* `I`.

If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead.